cmake/Modules/FindTPLCUDA.cmake

```
@@ -29,7 +29,7 @@ ELSE()
 ENDIF()
 
 include(FindPackageHandleStandardArgs)
-IF(KOKKOS_CXX_HOST_COMPILER_ID STREQUAL PGI)
+IF(KOKKOS_CXX_HOST_COMPILER_ID STREQUAL PGI OR KOKKOS_CXX_HOST_COMPILER_ID STREQUAL NVHPC)
```

```suggestion
IF(KOKKOS_CXX_HOST_COMPILER_ID STREQUAL NVHPC)
```
we overwrite PGI with NVHPC in `cmake/kokkos_compiler_id.cmake`, see https://github.com/kokkos/kokkos/blob/4ea41d7f872801af25c032cae868b8064b23281a/cmake/kokkos_compiler_id.cmake#L178-L181.

---

OK